alibi_detect/utils/fetching.py

```diff
@@ -511,3 +514,26 @@ def fetch_detector(filepath: Union[str, os.PathLike],
             kwargs = get_pixelcnn_default_kwargs()
     detector = load_detector(filepath, **kwargs)
     return detector
+
+
+def _join_url(base: str, parts: Union[str, List[str]]) -> str:
```

Comment:
Should we test this?

Reply:
Happy to leave as a TODO to get this merged asap, but the testing functionality wouldn't need to be very extensive as this is just for internal usage.